### PR TITLE
LITE-29258 Setup repo

### PR DESCRIPTION
**Pull request by jonatrios**

### CODE REVIEW

**Review by akodelia — APPROVED** [review]

**Review by maxipavlovic — APPROVED** [review]
In general, LGTM

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    name: Backend tests
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10']
```
Comment (maxipavlovic) [inline .github/workflows/test.yml:17]: We need only one 3.10 version :)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM cloudblueconnect/connect-extension-runner:27.25
```
[the remark this answers is not in the document]
Comment (jonatrios) [reply Dockerfile]: updated!

File: .github/workflows/test.yml (hunk 1 above)
Comment (jonatrios) [reply .github/workflows/test.yml]: updated!

**Review by maxipavlovic — APPROVED** [review]

**Pull request merged**